### ISSUE TITLE
CSSS picture with black text on a dark gray background in the footer

### ISSUE THREAD

**Issue by abejfehr** [abejfehr/CSSS#32]
This is annoying to look at, and is actually in violation of the [WCAG 2.0 guidelines](http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).

When I have time I'm gonna see if I can make the text white on the picture and use that instead.
